app/build.gradle

```diff
@@ -20,11 +20,12 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.support:appcompat-v7:27.1.1'
+    implementation 'com.android.support:appcompat-v7:27.1.0'
```

Comment:
Why did you change your dependency version codes from `.1` to `.0`

Reply:
I had to change the dependency version code from .1 to .0 because the Picasso **com.squareup.picasso:picasso:2.71828** internally depended on **v7:27.1.0** and all dependency must use the same dependency version code.

Reply:
@omokolataiwo 
Hmm... how strange!